circle.yml

```diff
@@ -566,7 +535,7 @@ jobs:
     steps:
       - run:
           name: Clone mapbox-gl-native-android in working directory
-          command: git clone https://github.com/mapbox/mapbox-gl-native-android.git .
+          command: git clone -b tarigo_pull_in_jni_bindings https://github.com/mapbox/mapbox-gl-native-android.git .
```

Comment:
This temporary workaround to get CI green, should be reverted back to master after mapbox/mapbox-gl-native-android#301 is merged.

Reply:
It's fine to merge the API breakage bot with a failing result, else it introduces a circular dependency and you won't be able to merge any of the two PRs. I prefer merging this PR first and updating downstream PR to master of gl-native.

Reply:
Reverted back to master.